rdrf/rdrf/tests.py

```diff
@@ -485,3 +486,569 @@ def test_date(self):
     def test_nonascii(self):
         err = check_calculation("context.result = '💩';")
         self.assertEqual(err, "")
+
+
+
+class FakeModjgo(object):
+            def __init__(self, pk, data):
+                self.pk = pk
+                self.data = data
+            def save(self):
+                print("Fake Modjgo save called")
+
+
+class TimeStripperTestCase(TestCase):
+    def setUp(self):
+        super(TimeStripperTestCase, self).setUp()
```

Comment:
The super(CLASS, self).x(...) calls can all be changed to super().x(...) now that we're on python 3. 

Applies to all the similar calls.